noobaa/noobaa-core
# move phone-home-form to new arch

[skip ci]

### Issues: Fixed #xxx / Gap #xxx
- move phone-home-form to new arch
- API renamed

### Testing Instructions:
1. go to management
2. expand Proxy Address
3. switch on remote Proxy Address
4. click save, validation errors displayed correctly
5. type address '1.1.1.1' and port '80'
6. dirty mark is shown near the header
7. click Apply Change
8. info notification displayed: 'Checking external services connectivity using configure proxy'
9. working buttons in progress
10. the test failed and notification displayed: 'External services could not be reached using configured proxy'

### Reminders:
- Create a new test - the first is the hardest
- User Experience is top priority
- Design for failure
- Keep it simple
- `npm test`
- Imagine a support call - Add diagnostics info, phonehome info, activity log events, external syslog
- Upgrade - DB schema, server platform, agents install, npm packages
